operator/api/redpanda/v1alpha2/zz_generated_status.go

```diff
@@ -0,0 +1,258 @@
+
+// "ResourcesSynced" - This condition indicates whether the Kubernetes resources
+// for a cluster have been synchronized.
+//
+// This condition defaults to "False" with a reason of "NotReconciled" and must
+// be set by a controller when it subsequently reconciles a cluster.
+// +statusType
+type ClusterResourcesSyncedCondition string
+
+// "ConfigurationApplied" - This condition indicates whether cluster
+// configuration parameters have currently been applied to a cluster for the
+// given generation.
+//
+// This condition defaults to "False" with a reason of "NotReconciled" and must
+// be set by a controller when it subsequently reconciles a cluster.
+// +statusType
+type ClusterConfigurationAppliedCondition string
+
+// "Quiesced" - This condition is used as to indicate that the cluster is no
+// longer reconciling due to it being in a finalized state for the current
+// generation.
+//
+// This condition defaults to "False" with a reason of "NotReconciled" and must
+// be set by a controller when it subsequently reconciles a cluster.
+// +statusType
+type ClusterQuiescedCondition string
+
+// "Stable" - This condition is used as a roll-up status for any sort of
+// automation such as terraform.
+//
+// This condition defaults to "False" with a reason of "NotReconciled" and must
+// be set by a controller when it subsequently reconciles a cluster.
+// +statusType
+type ClusterStableCondition string
+
+// "Bound" - This condition indicates whether a node pool is bound to a known
+// Redpanda cluster.
+//
+// This condition defaults to "Unknown" with a reason of "NotReconciled" and
+// must be set by a controller when it subsequently reconciles a node pool.
+// +statusType
+type NodePoolBoundCondition string
+
+// "Deployed" - This condition indicates whether a node pool has been deployed
+// for a known Redpanda cluster.
+//
+// This condition defaults to "Unknown" with a reason of "NotReconciled" and
+// must be set by a controller when it subsequently reconciles a node pool.
+// +statusType
+type NodePoolDeployedCondition string
+
+// "Quiesced" - This condition is used as to indicate that the node pool is no
+// longer reconciling due to it being in a finalized state for the current
+// generation.
+//
+// This condition defaults to "False" with a reason of "NotReconciled" and must
+// be set by a controller when it subsequently reconciles a node pool.
+// +statusType
+type NodePoolQuiescedCondition string
+
+// "Stable" - This condition is used as a roll-up status for any sort of
+// automation such as terraform.
+//
+// This condition defaults to "False" with a reason of "NotReconciled" and must
+// be set by a controller when it subsequently reconciles a node pool.
+// +statusType
+type NodePoolStableCondition string
+
+const (
+	// This reason is used with the "Ready" condition when it evaluates to True
+	// because a cluster can service traffic.
+	ClusterReadyReasonReady ClusterReadyCondition = "Ready"
```

Comment:
For external consumption should these be prefixed with `Redpanda` to match the convention of aligning with CRD names? That might be Redpanda overload though...